telosnetwork/telos-wallet
# #168 | App skeleton

# Fixes #168

## Description
This PR adds in the overall layout for the EVM pages, nav/menu, and a standardized way to structure pages. 

## Test scenarios
- go to https://deploy-preview-218--wallet-staging.netlify.app/

- 'log in'  by clicking Connect Your Wallet
    - you should be taken to the new evm wallet route
- click around on the nav and page tabs
    - nav styling should update depending on your route
    - page layout and tabs should match the designs
- use your phone or devtools to view the mobile site
    - menu should collapse to hamburger icon
    - mobile menu should match designs



## Checklist:
